File: benchmarks/bench_primes.py
import time, sys
def is_prime(n):
    if n < 2: return False
    d = 2
    while d * d <= n:
        if n % d == 0: return False
        d += 1
    return True

t0 = time.perf_counter()
result = 0
for r in range(30):
    count = 0
    for n in range(2, 100001):
        if is_prime(n): count += 1
    result = count
ms = (time.perf_counter() - t0) * 1000
print(result)
print(f'COMPUTE_MS={ms:.3f}', file=sys.stderr)

File: benchmarks/bench_primes.go
package main
import ("fmt"; "os")
func is_prime(n int) bool {
	if n < 2 { return false }
	for d := 2; d*d <= n; d++ { if n%d == 0 { return false } }
	return true
}
func main() {
	result := 0
	for r := 0; r < 30; r++ {
		count := 0
		for n := 2; n <= 100000; n++ { if is_prime(n) { count++ } }
		result = count
	}
	fmt.Println(result)
	os.Exit(0)
}
